codeSnippets/snippets/gradle-app/kotlin/app.kt

```diff
@@ -0,0 +1,18 @@
+// src/main/kotlin/app.kt
+
+import io.ktor.application.*
+import io.ktor.response.*
+import io.ktor.routing.*
+import io.ktor.server.cio.*
+import io.ktor.server.engine.*
+
+fun main() {
+    val server = embeddedServer(CIO, port = 8080) {
```

Comment:
Not sure if we should be using CIO, especially since we don't recommend it yet for production.  Also, we should probably avoid the embeddedServer and use HOCON like most other snippets.

Reply:
The goal of this article is to show how to package an application and the choice of an engine isn't relevant here. Again, I see no difference between the usage of `embeddedServer` and `HOCON` here, so I chose the simplest one.

Reply:
I mention this for two reasons:

1. I think it would make sense to have consistency across our code snippets
2. In real world applications, people need to deploy other files, and resources is a good example. In fact I'd go as far as saying your sample should include examples of using `static` too so that people understand where these files are copied and distributed to.

Reply:
1. We have two up-to-date methods. Why in the documentation should I prefer only one? 
2. I agree about showing packing resources but that's not about `HOCON`

Reply:
Not about preference, but about consistency. In any case no objections, but would be good to show then examples with static.

Reply:
I also vote for consistency:
1. This example should be runnable.
2. Have the same structure as other examples from the `snippets` folder.
3. Have readme. 
You can still keep the initial Gradle file and include Ktor dependencies using the `lines` attribute (it works now).